feature/networkinstance/networkinstance_test.go

```diff
@@ -85,6 +90,117 @@ func TestAugment(t *testing.T) {
 				},
 			},
 		},
+	}, {
+		desc:     "Static route",
+		ni:       New("default", oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE),
+		inDevice: &oc.Device{},
+		wantDevice: &oc.Device{
+			NetworkInstance: map[string]*oc.NetworkInstance{
+				"default": {
+					Name:    ygot.String("default"),
+					Enabled: ygot.Bool(true),
+					Type:    oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE,
+					Protocol: map[oc.NetworkInstance_Protocol_Key]*oc.NetworkInstance_Protocol{
+						protocolKey: {
+							Identifier: oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC,
+							Name:       ygot.String("static"),
+							Static: map[string]*oc.NetworkInstance_Protocol_Static{
+								"primary": {
+									Prefix: ygot.String("1.1.1.1"),
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+	}, {
+		desc:     "One Next Hop",
+		ni:       New("default", oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE),
+		inDevice: &oc.Device{},
+		wantDevice: &oc.Device{
+			NetworkInstance: map[string]*oc.NetworkInstance{
+				"default": {
+					Name:    ygot.String("default"),
+					Enabled: ygot.Bool(true),
+					Type:    oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE,
+					Protocol: map[oc.NetworkInstance_Protocol_Key]*oc.NetworkInstance_Protocol{
+						protocolKey: {
+							Identifier: oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC,
+							Name:       ygot.String("static"),
+							Static: map[string]*oc.NetworkInstance_Protocol_Static{
+								"primary": {
+									NextHop: map[string]*oc.NetworkInstance_Protocol_Static_NextHop{
+										"0": {
+											NextHop: oc.UnionString("1.2.3.44"),
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+	}, {
+		desc:     "Zero Next Hop",
```

Comment:
this is not useful test. How about "Static route with multiple next-hops". You can then supply say 2 NHs.

Reply:
Multiple next hops are already added below. Will remove this test as "Static Route with no next Hop" essentially tests this